<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StructureViewSettingsWidget</class>
 <widget class="QWidget" name="StructureViewSettingsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>411</width>
    <height>292</height>
   </rect>
  </property>
  <layout class="QFormLayout" name="formLayout_2">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="fieldGrowthPolicy">
    <enum>QFormLayout::ExpandingFieldsGrow</enum>
   </property>
   <item row="0" column="0">
    <widget class="QLabel" name="byteOrderLabel">
     <property name="text">
      <string>Byte order:</string>
     </property>
     <property name="buddy">
      <cstring>kcfg_ByteOrder</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="KComboBox" name="kcfg_ByteOrder">
     <property name="sizeAdjustPolicy">
      <enum>QComboBox::AdjustToContents</enum>
     </property>
     <item>
      <property name="text">
       <string>Big endian</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Little endian</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QCheckBox" name="kcfg_ShortTypeNames">
     <property name="text">
      <string>Use short type names</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QCheckBox" name="kcfg_LocaleAwareDecimalFormatting">
     <property name="text">
      <string>Use locale-aware integer formatting</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="unsigned_label">
     <property name="text">
      <string>Unsigned values:</string>
     </property>
     <property name="buddy">
      <cstring>combo_UnsignedDisplayBase</cstring>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="KComboBox" name="combo_UnsignedDisplayBase"/>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="signed_label">
     <property name="text">
      <string>Signed values:</string>
     </property>
     <property name="buddy">
      <cstring>combo_SignedDisplayBase</cstring>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="KComboBox" name="combo_SignedDisplayBase"/>
   </item>
   <item row="5" column="0" colspan="2">
    <widget class="QCheckBox" name="kcfg_LocaleAwareFloatFormatting">
     <property name="text">
      <string>Use locale-aware floating-point formatting</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Floating-point precision:</string>
     </property>
     <property name="buddy">
      <cstring>kcfg_FloatPrecision</cstring>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QSpinBox" name="kcfg_FloatPrecision"/>
   </item>
   <item row="7" column="0" colspan="2">
    <widget class="QCheckBox" name="kcfg_ShowCharNumericalValue">
     <property name="text">
      <string>Show numerical value of chars</string>
     </property>
    </widget>
   </item>
   <item row="8" column="0">
    <widget class="QLabel" name="char_label">
     <property name="text">
      <string>Character value:</string>
     </property>
     <property name="buddy">
      <cstring>combo_CharDisplayBase</cstring>
     </property>
    </widget>
   </item>
   <item row="8" column="1">
    <widget class="KComboBox" name="combo_CharDisplayBase"/>
   </item>
   <item row="9" column="0">
    <widget class="QSpinBox" name="kcfg_CharDisplayBase"/>
   </item>
   <item row="9" column="1">
    <widget class="QSpinBox" name="kcfg_SignedDisplayBase"/>
   </item>
   <item row="10" column="0">
    <widget class="QSpinBox" name="kcfg_UnsignedDisplayBase"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KComboBox</class>
   <extends>QComboBox</extends>
   <header>kcombobox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>kcfg_ShowCharNumericalValue</sender>
   <signal>toggled(bool)</signal>
   <receiver>combo_CharDisplayBase</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>201</x>
     <y>166</y>
    </hint>
    <hint type="destinationlabel">
     <x>388</x>
     <y>194</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
